### PR TITLE
Updated wording on FAQ

### PR DESCRIPTION
**Pull request by OliverCollins**
## What this PR does / why we need it

A few sentences describing the overall goals of the pull request's commits.

## Related Issues

Fixes #

## Some questions

- [x] I have read the contributing guidelines
- [x] I abide by this repository Code of Conduct

## Special notes for your reviewer

(optional)

## Additional Notes

Do you want to add anything else? We :heart: to hear your opinions!

### CODE REVIEW

File: _data/faq.yaml [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+- question: What should I bring?
+  answer: "In order to check-in, you must have a government-issued ID. But, you should also bring your laptop, laptop charger, phone, chargers, headphones, a pillow, toiletries, an extra set of clothes, and a reusable water bottle."
 - question: Will you offer travel reimbursement?
-  answer: "YES! Stay tuned for more details"
+  answer: "Yes! Travel reimbursement will be based on a case-by-case basis. Stay tuned for more details!"
 - question: What if I am not a student?
   answer: "Be a volunteer or mentor! More information coming soon!"
 - question: Do I have to be a programmer?
-  answer: "Absolutely not! It doesn’t matter if you are a designer, a writer, or a chemistry major. There is a place for you here. There will be a team formation session at the beginning of the event to help form teams."
+  answer: "Absolutely not! It doesn’t matter if you are a designer, a writer, or a chemist. There is a place for you here. Prior to hacking, there will be a team formation session at the beginning of the event to help form teams."
```
Comment (AparaV) [inline _data/faq.yaml]: `Prior to hacking, [...] at the beginning of the event [...]` - Can you remove one of those?

**Review by AparaV — CHANGES_REQUESTED** [review]
Also, adding a description about what this PR does helps! 🙃

**Comment by casassg** [comment]
Also, I would appreciate if you fill the description of the PR next time @OliverCollins

File: _data/faq.yaml [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 - question: Will there be food?
-  answer: "Yes, lots! We will provide three meals a day as well as snacks, drinks, and caffeine."
-- question: What do I bring?
-  answer: "A laptop, laptop charger, phone, chargers, headphones, a pillow, toiletries, an extra set of clothes, and a water bottle."
+  answer: "Plenty, we will provide three meals a day as well as snacks, drinks, and caffeine."
+- question: What should I bring?
+  answer: "In order to check-in, you must have a government-issued ID. But, you should also bring your laptop, laptop charger, phone, chargers, headphones, a pillow, toiletries, an extra set of clothes, and a reusable water bottle."
 - question: Will you offer travel reimbursement?
-  answer: "YES! Stay tuned for more details"
+  answer: "Yes! Travel reimbursement will be based on a case-by-case basis. Stay tuned for more details!"
```
Comment (casassg) [inline _data/faq.yaml]: Also, this needs to be specified at next meeting.

**Review by casassg — CHANGES_REQUESTED** [review]
Holding this at least until next meeting to decide on this. Otherwise, it should be removed to a more agnostic answer

**Review by casassg — APPROVED** [review]
updated to have it merged myself

**Review by AparaV — APPROVED** [review]

**Pull request merged**